<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<title>javascript中内建对象之error对象</title>
		<script type="text/javascript">
			try{
				notExitsts();
			}catch(e){
				if(e instanceof EvalError){
					alert('EvalError'+ e.name+ e.message)
				}elif(e instanceof RangeError){
					alert('RangeError'+ e.name+ e.message)
				}
			}finally{
				alert('总是被调用')
			}
			
			
			
			
		</script>

		
		
		
		
	</head>
	<body>
		<h2>javascript中内建对象之error对象</h2>
	</body>
</html>
